non-atomic-action-interpreter-module/README.md

```diff
@@ -0,0 +1,135 @@
+# Non-Atomic Action Interpreter Module
+
+This module is part of the ostis-ps-lib and is responsible for interpreting non-atomic actions within the OSTIS.
+
+## Using module as an Extension for sc-machine
+
+The non-atomic action interpreter Module can be used as an extension to the sc-machine. Follow these steps to integrate it:
+
+### 1. Download sc-machine Artifacts
+
+   - Download pre-built artifacts of sc-machine from [GitHub Releases](https://github.com/ostis-ai/sc-machine/releases) and extract them to a location of your choice.
+   - Alternatively, you can build sc-machine from sources and use the resulting artifacts.
+   - For more details on installing and using sc-machine artifacts, refer to the [sc-machine quick start guide](https://ostis-ai.github.io/sc-machine/quick_start/).
+  
+### 2. Download Non-Atomic Action Interpreter Module Artifacts
+
+   - Download pre-built artifacts of Non-Atomic Action Interpreter Module from [GitHub Releases](https://github.com/ostis-ai/ostis-ps-lib/releases) and extract them to a location of your choice.
```

Comment:
is it possible to download only one module from release?

Reply:
Course. You can try to do this in archives of sc-machine, for example

Reply:
I see only whole sc-machine option
![image](https://github.com/user-attachments/assets/47232ddc-cf00-4f62-b07d-40f4357c1b9e)

Reply:
There will be several archives in releases of this repository, each of which can be downloaded